src/ngx_http_lua_log_ringbuf.c

```diff
@@ -0,0 +1,201 @@
+
+#ifndef DDEBUG
+#define DDEBUG 0
+#endif
+#include "ddebug.h"
+
+
+#include "ngx_http_lua_common.h"
+#include "ngx_http_lua_log_ringbuf.h"
+
+
+typedef struct {
+    unsigned    len:24;
+    unsigned    log_level:4;
```

Comment:
@membphis Maybe we should make it fill 32-bit instead of 28-bit here? This struct will be 32-bit due to padding requirements anyway.